packages/klevu-ui/src/utils/getTranslation.ts

```diff
@@ -0,0 +1,33 @@
+import en from "../translations/en.json"
+
+type TranslationKeys = typeof en
+
+type Join<K, P> = K extends string | number
+  ? P extends string | number
+    ? `${K}${"" extends P ? "" : "."}${P}`
+    : never
+  : never
+
+type Paths<T, D extends number = 10> = [D] extends [never]
+  ? never
+  : T extends object
+  ? {
+      [K in keyof T]-?: K extends string | number ? `${K}` | Join<K, Paths<T[K], any>> : never
+    }[keyof T]
+  : ""
+
+export function getTranslation(key: Paths<TranslationKeys, 10>) {
+  const keys = key.split(".")
+  let value = window.klevu_ui_translations || en
+
+  for (const key of keys) {
+    if (value.hasOwnProperty(key)) {
+      // @ts-ignore
+      value = value[key]
+    } else {
+      throw new Error(`Translation "${key}" not found`)
```

Comment:
would it better to just return key itself when not found, wont break anything. error can be printed to console.error

Reply:
Make change by request